src/main/java/nextstep/subway/application/StationService.java

```diff
@@ -25,12 +27,18 @@ public StationResponse saveStation(StationRequest stationRequest) {
         return StationResponse.of(persistStation);
     }
 
+    public Station findById(Long id) {
+        return stationRepository.findById(id)
+            .orElseThrow(() -> new NoSuchElementException("해당 지하철역을 찾을 수 없습니다."));
+    }
+
+
     public List<StationResponse> findAllStations() {
         List<Station> stations = stationRepository.findAll();
 
         return stations.stream()
-                .map(station -> StationResponse.of(station))
-                .collect(Collectors.toList());
+            .map(station -> StationResponse.of(station))
```

Comment:
여기도 method reference 사용할 수 있을 것 같습니다!
```suggestion
            .map(StationResponse::of)
```